opencog/guile/SchemeSmobTV.cc

```diff
@@ -460,5 +460,32 @@ SCM SchemeSmob::ss_tv_get_value (SCM s)
 	return SCM_EOL;
 }
 
+/**
+ * Return the truth value mean
+ */
+SCM SchemeSmob::ss_tv_get_mean(SCM s)
+{
+	TruthValue *tv = verify_tv(s, "cog-tv-mean");
+	return scm_from_double(tv->getMean());
+}
```

Comment:
I mean, if you want to merge this branch, do it, this is not a show-stopper.  Just in general, I would prefer not to glom up the C++ API with useless code that would be better done in scheme, directly, where you'll get better performance (since the scheme code is compiled! (into bytecode) whereas the C++ code requires three slow and clunky calls across the guile-C++ interface. The bytecode will run much faster than two or three API calls could -- approximately 10x or 20x faster, based on benchmark measurements in the /opencog/benchmark directory)